static/absolute/js/searchPage.js

```diff
@@ -105,14 +104,15 @@ socket.on("refresh", () => {
 
 async function searchRestaurants() {
```

Comment:
Maybe rename this function to `refreshUI`, since it's not really just searching for restaurants.

Reply:
Alright

Reply:
done